docker/development/Dockerfile

```diff
@@ -6,10 +6,9 @@ COPY pdfcomprezzor/go.mod pdfcomprezzor/go.sum pdfcomprezzor/main.go ./
 RUN GOOS=js GOARCH=wasm go build -o pdfcomprezzor.wasm
 RUN cp "$(go env GOROOT)/misc/wasm/wasm_exec.js" .
 
-# Now build the actual mampf application
+# Now build the actual MaMpf application
 # https://hub.docker.com/_/ruby/
 FROM ruby:3.1.4-bullseye
-ENV RAILS_ENV=production
```

Comment:
I'm really not sure why we set the environment to _production_ beforehand in our development dockerfile. Fixed this and  I hope this doesn't have any unintended side-effects for our local setup.